libkbfs/folder_branch_ops.go

```diff
@@ -3766,17 +4009,41 @@ func (fbo *folderBranchOps) UnregisterFromChanges(obs Observer) error {
 	return nil
 }
 
-// notifyBatchLocked sends out a notification for the most recent op
-// in md.
+// notifyBatchLocked sends out a notification for all the ops in md.
 func (fbo *folderBranchOps) notifyBatchLocked(
 	ctx context.Context, lState *lockState, md ImmutableRootMetadata,
 	afterUpdateFn func() error) error {
 	fbo.headLock.AssertLocked(lState)
 
-	lastOp := md.data.Changes.Ops[len(md.data.Changes.Ops)-1]
-	err := fbo.notifyOneOpLocked(ctx, lState, lastOp, md, false, afterUpdateFn)
-	if err != nil {
-		return err
+	switch len(md.data.Changes.Ops) {
+	case 0:
+		return nil
```

Comment:
Should this ever happen? In the old code this would panic, right? Maybe we should keep that behavior.